plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/ReplaceableSelectionProvider.java

```diff
@@ -19,65 +19,76 @@
 
 /**
  * Since a MapEditor can have a different selection provider depending on the tool that is selected, this class
- * is needed to manage the changing selection providers.  
- * 
+ * is needed to manage the changing selection providers.
+ *
  * @author Jesse
  * @since 1.1.0
  */
-class ReplaceableSelectionProvider implements ISelectionProvider {
+public class ReplaceableSelectionProvider implements ISelectionProvider {
```

Comment:
should it be visible from outside of the package?

Reply:
I am using this class in a self-implemented ViewPart therefore I need to access it :)